<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SalesDateDialog</class>
 <widget class="QDialog" name="SalesDateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>232</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>232</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sales Dates</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel,QDialog,QDateEdit,QPushButton{
color:white;
background-color:rgb(54,54,54);
	font: 12pt &quot;MS Shell Dlg 2&quot;;
}
QPushButton{
border:1px solid grey;
border-radius:5px;
}
QPushButton:hover{
border:2px solid white;
}
QPushButton:pressed{
border:3px solid grey;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="onWdg" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Choose a date to view the sales</string>
        </property>
        <property name="buddy">
         <cstring>dateEdit_salesOn</cstring>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Sales On : </string>
          </property>
          <property name="buddy">
           <cstring>dateEdit_salesOn</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit_salesOn">
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>35</height>
           </size>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="periodWdg" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Choose the sales period dates </string>
        </property>
        <property name="buddy">
         <cstring>dateEdit_salesFro</cstring>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>From : </string>
          </property>
          <property name="buddy">
           <cstring>dateEdit_salesFro</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit_salesFro">
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>35</height>
           </size>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string> To : </string>
          </property>
          <property name="buddy">
           <cstring>dateEdit_salesTo</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit_salesTo">
          <property name="minimumSize">
           <size>
            <width>125</width>
            <height>35</height>
           </size>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(197, 131, 98);</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openBtn">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(56, 168, 82);</string>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
